### PR TITLE
Add storeAttribute and storeXPathCount commands

### PR DESCRIPTION
**Pull request by Plotist**
By request from https://github.com/SeleniumHQ/selenium-ide/issues/160 and https://github.com/SeleniumHQ/selenium-ide/issues/154

### CODE REVIEW

**Comment by corevo** [comment]
I'm aware of the bug that failed your tests, for the time being change the locator from ```//button``` to ```xpath=//button```.  
It is fixed along with the assertion refactor [here](https://github.com/SeleniumHQ/selenium-ide/pull/153)

**Comment by Plotist** [comment]
@corevo thanks! it was kind of confusing

File: packages/selianize/src/command.js [hunk 1]
```diff
@@ -202,6 +204,18 @@ async function emitStoreTitle(_, varName) {
...
+async function emitStoreXpathCount(locator, varName) {
+  return Promise.resolve(`var ${varName};driver.wait(until.elementsLocated(${await LocationEmitter.emit(locator)}));driver.findElements(${await LocationEmitter.emit(locator)}).then(elements => {${varName} = elements.length;});`);
+}
+
+async function emitStoreAttribute(locator, varName) {
+  const attributePos = locator.lastIndexOf("@");
+  const elementLocator = locator.slice(0, attributePos);
+  const attributeName = locator.slice(attributePos + 1);
+
+  return Promise.resolve(`var ${varName};driver.wait(until.elementLocated(${await LocationEmitter.emit(elementLocator)})); driver.findElement(${await LocationEmitter.emit(elementLocator)}).then(element => element.getAttribute("${attributeName}").then(attribute => {${varName} = attribute;}));`);
```
Comment (corevo) [inline packages/selianize/src/command.js]: The use of `var` is discouraged, to enable users to redefine it use `let` instead, both will be defined in the `it` block.

**Comment by Plotist** [comment]
@corevo is it ready to be merged now?

**Pull request merged**